updatecli/internal/02_proxy.yml

```diff
@@ -0,0 +1,48 @@
+name: Bump Proxy Chart Version
+sources:
+  currentProxyVersion:
+    kind: githubrelease
+    spec:
+      owner: "observeinc"
+      repository: "helm-charts"
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      versionFilter:
+        kind: regex
+        pattern: "proxy-(.*)$"
+    transformers:
+      - trimprefix: "proxy-"
+conditions:
+  chartChanged:
+    name: "Check if charts/proxy/ has changes"
+    kind: shell
+    spec:
+      command: git diff --quiet --name-only origin/main...HEAD -- charts/proxy/ && git diff --quiet --name-only -- charts/proxy/ && exit 1 || exit 0
+    disablesourceinput: true
+targets:
+  updateProxyChart:
+    name: "Bump Patch Version"
+    kind: yaml
+    sourceid: currentProxyVersion
+    transformers:
+      - semverinc: "patch"
+    spec:
+      file: "charts/proxy/Chart.yaml"
+      key: "$.version"
+  updateStackChart:
+    name: "Update Stack Chart Dependency"
+    kind: yaml
+    sourceid: currentProxyVersion
+    transformers:
+      - semverinc: "patch"
+    spec:
+      file: "charts/stack/Chart.yaml"
+      key: "$.dependencies[3].version"
```

Comment:
a bug in the yaml parsing library means we must use array indexes as it's not possible to search based on `:name=xxx`